contracts/FiatToken.sol

```diff
@@ -1,9 +1,9 @@
 pragma solidity ^0.5.0;
 
-import 'openzeppelin-solidity/contracts/token/ERC20/ERC20.sol';
-import 'openzeppelin-solidity/contracts/token/ERC20/ERC20Detailed.sol';
-import 'openzeppelin-solidity/contracts/token/ERC20/ERC20Mintable.sol';
-import 'openzeppelin-solidity/contracts/token/ERC20/ERC20Burnable.sol';
+import "../node_modules/openzeppelin-solidity/contracts/token/ERC20/ERC20.sol";
```

Comment:
This is not needed. Is solved using `.vscode/settings.json` on master branch

Idea from: https://medium.com/coinmonks/solidity-import-in-vs-code-the-right-way-82baa1cc5a71